pkg/columns/formatter/textcolumns/scaler.go

```diff
@@ -58,46 +62,88 @@ func (tf *TextColumnsFormatter[T]) RecalculateWidths(maxWidth int, force bool) {
 	tf.currentMaxWidth = maxWidth
 
 	// Get total width of all printed columns
-	totalWidth := 0
-	spaces := 0
+	totalWidthNotFixed := 0
+	totalWidthFixed := 0
 	for i, column := range tf.showColumns {
 		if i > 0 {
-			spaces += len([]rune(tf.options.ColumnDivider))
+			totalWidthFixed += len([]rune(tf.options.ColumnDivider))
```

Comment:
Why do you have to use `len([]rune(tf.options.ColumnDivider))` instead of `len(tf.options.ColumnDivider)`?

Reply:
We have to use []rune, because the string ColumnDivider could contain UTF encoded characters that would count as more characters than actually printed.

Reply:
TIL. Thanks!